src/coreComponents/physicsSolvers/fluidFlow/integratedTests/singlePhaseFlow/fractureMatrixFlow_Embedded2d.xml

```diff
@@ -24,7 +24,8 @@
       logLevel="1"
       solidMaterialNames="{ rock }"
       targetRegions="{ Region2, Fracture }"
-      fractureRegion="Fracture"/>
+      fractureRegion="Fracture"
+      mpiCommOrder="1"/>
```

Comment:
what is mpiCommOrder?

Reply:
This is exactly the same as in the `SurfaceGenerator`.  When syncing the embedded surfaces between ranks it forces the code to process the ranks in order. If I don't do that I randomly fail the integratedTests. I believe that's because the local numbering of the ghosts on each rank will vary depending on which ghosts it receives first.